content/en/docs/instrumentation/js/automatic/_index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Automatic Instrumentation
+linkTitle: Automatic
+weight: 30
+---
+
+Automatic instrumentation provides a way to instrument any Node.js application
+and capture telemetry data from many popular libraries and frameworks without
+any code changes.
+
+## Setup
+
+Run the following commands to install the appropriate packages.
+
+```shell
+npm install --save @opentelemetry/api
+npm install --save @opentelemetry/auto-instrumentations-node
+```
+
+The `@opentelemetry/api` and `@opentelemetry/auto-instrumentations-node`
+packages install the API, SDK, and the instrumentation tools.
+
+## Configuring the module
+
+The module is highly configurable.
+
+One option is to configure the module by way of using `env` to set environment
+variables from the CLI:
+
+```shell
+env OTEL_TRACES_EXPORTER=otlp OTEL_EXPORTER_OTLP_TRACES_ENDPOINT=your-endpoint \
+node --require @opentelemetry/auto-instrumentations-node/register app.js
+```
+
+Alternatively, you can use `export` to set environment variables:
+
+```shell
+export OTEL_TRACES_EXPORTER="otlp"
+export OTEL_EXPORTER_OTLP_TRACES_ENDPOINT="your-endpoint"
```

Comment:
Would it make sense to use the `OTEL_EXPORTER_OTLP_ENDPOINT` env var instead, so that it covers metrics as well?

Reply:
Updated :)
Having the broader env var makes more sense.